File: Length_of_Last_Word/lengthOfLastWord.java
public class lengthOfLastWord {
    public static void main(String[] args) {
        process("Hello World");
        process("Luffy is still joyboy");
        process("Luffy is gonna be the king of pirates");
    }

    private static void process(String text) {
        String lastWord = "";
        String[] words = text.split(" ");
        for (String word: words) {
            lastWord = words[words.length-1];
        }
        int lengthLastWord = lastWord.length();
        System.out.println("The last word is " + lastWord + "with length " + lengthLastWord + ".");
    }
}

File: Length_of_Last_Word/lengthOfLastWord.py
def length_of_last_word(text):
    words=text.split()
    last_word = words[-1]
    length_last_word = len(last_word)
    print(f"The last word is {last_word} with length {length_last_word}.")

length_of_last_word("Hello World")
length_of_last_word("Fly me to the moon")
length_of_last_word("Luffy is still joyboy")
length_of_last_word("Luffy is the king of pirates")
